packages/postgres-database/src/simcore_postgres_database/models/groups_extra_properties.py

```diff
@@ -63,6 +63,13 @@
         server_default=sa.sql.expression.false(),
         doc="If true, will send telemetry for new style dynamic services to frontend",
     ),
+    sa.Column(
+        "enable_efs",
```

Comment:
Q: I thought we wanted to enable this only for a certain subset of users and not for all everybody.

Reply:
Yes and this table is exactly for that, for a specific group you can enable it.